onnxruntime/test/providers/cpu/signal/signal_ops_test.cc

```diff
@@ -33,6 +33,10 @@ static void TestNaiveDFTFloat(bool onesided, int since_version) {
     expected_output.resize(6);
   }
   test.AddInput<float>("input", shape, input);
+  if (since_version == 20) {
```

Comment:
does this still test opset 17 (in addition to 20)?

Reply:
The test is for both versions, the new inputs are only tested in version 20.